module/sell.py

```diff
@@ -1,62 +1,57 @@
+                    price = str(format(float(message.split()[2].replace(',', '.')), '.2f'))
+                    context.bot.send_message(chat_id, "Ricerca del libro associato all'ISBN inserito...")
+                    
+                    books = get_books()
+                    found, n = find_book(user_isbn, books)
+                    if found:
+                        isbn = str(books.iloc[n][0])
+                        title  = str(books.iloc[n][1])
+                        authors  = str(books.iloc[n][2])
+                        title_and_authors  = title + "di" + authors
+                        context.bot.send_message(chat_id, "Il libro è: \nTitolo: \"" + title_and_authors + "\"")
+                        add_item(isbn, title, authors, username, price)
+                        context.bot.send_message(chat_id, "Il libro è stato aggiunto al database.")
+                        return
+                    
+                    found, soup = book_in_unict(user_isbn)
+                    if found:
+                        isbn = str(soup.findAll('td')).split("bibInfoData")[len(str(soup.findAll('td')).split("bibInfoData"))-1].split("\n")[1].split("<")[0]
                         title  = soup.find('strong').text.split('/')[0]
                         authors  = soup.find('strong').text.split('/')[1]
-                        isbn = str(soup.findAll('td')).split("bibInfoData")[len(str(soup.findAll('td')).split("bibInfoData"))-1].split("\n")[1].split("<")[0]
-                        df = get_on_sale()
-                        new_row = pd.DataFrame({'ISBN':str(isbn),'Titolo':str(title), 'Autori':str(authors), 'Venditore':str(username), 'Prezzo':str(prezzo)}, index=[len(df)])
-                        df = pd.concat([df, new_row])
-                        df.to_csv(SHOP_DB_PATH)
-                        new_book = pd.DataFrame({'ISBN':str(isbn),'Titolo':str(title), 'Autori':str(authors)}, index=[len(books)])
-                        books = pd.concat([books, new_book])
-                        books.to_csv(BOOKS_DB_PATH)
+                        title_and_authors  = soup.find('strong').text
+                        context.bot.send_message(chat_id, "Il libro è: \nTitolo: \"" + title_and_authors + "\"")
+                        add_item(isbn, title, authors, username, price)
+                        add_book(isbn, title, authors)
                         context.bot.send_message(chat_id, "Il libro è stato aggiunto al database.")
-                    else:
-                        context.bot.send_message(chat_id, "Libro non trovato. Controlla di aver inserito correttamente l'ISBN.")
-            except Exception as e:
-                print(str(e))
-                context.bot.send_message(chat_id, "Prezzo non valido.")
+                        return
+                    
+                    context.bot.send_message(chat_id, "Libro non trovato. Controlla di aver inserito correttamente l'ISBN.")
+                    return
```

Comment:
I don't think that this is important, when a function end by default return None

Reply:
Right, I forgot to remove that